lib/models/player.rb

```diff
@@ -0,0 +1,16 @@
+module RubyConnect
+  class Player
+    attr_reader :game, :color
+
+    def initialize(info = {})
+      @name         = info[:name]
+      @phone_number = info[:phone_number]
+      @color        = info[:color]
+      @game         = info[:game]
```

Comment:
Is this a game object or a game id? Does a game really belong to the player? What if we want a player to be able to paly multiple games at the same time?